uswds/uswds
# [Identifier] Update Accessibility statement language

# Summary

**Updated Accessibility statement link text.** Updated the Identifier to use the text "Accessibility statement" (EN) / "Declaración de accesibilidad" (ES) for the required link to an accessibility statement.

## Breaking change

:warning: This is a breaking change because it changes the content of the Identifier. Earlier versions of the Identifier will continue to work as expected.

Teams should update their Identifiers to use "Accessibility statement" for the link text that links to their accessibility statement. Use "Declaración de accesibilidad" for Spanish-language versions of the Identifier.

## Related issue

Closes #5092 

## Related pull requests

- [x] Guidance and changelog: https://github.com/uswds/uswds-site/pull/2097

## Preview link

[Preview link](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/dw-identifier-accessibility/?path=/story/components-identifier--default) → 

## Problem statement

Our Identifier should support the OMB memorandum ["Strategic Plan for Improving Management of Section 508 of the Rehabilitation Act"](https://obamawhitehouse.archives.gov/sites/default/files/omb/procurement/memo/strategic-plan-508-compliance.pdf) [PDF, 212 KB] by linking to their agency's accessibility statement. Section508.gov has [more information about the accessibility statement](https://www.section508.gov/manage/laws-and-policies/website-accessibility-statement/). This accessibility statement includes Section 508 program and contact information, but should not be considered "accessibility support" in the conventional user-facing understanding.

Our guidance (and [the required links guidance on Digital.gov](https://digital.gov/resources/required-web-content-and-links/#accessibility-statement)) indicates that this link should be called `Accessibility` but the Identifier currently uses `Accessibility support`. 

Since this link is a link to an accessibility statement, our Identifier should use `Accessibility statement`.

## Solution

This PR updates the `Accessibility` link to `Accessibility statement`. It updates the Spanish-language version from `Accesibilidad` to `Declaración de accesibilidad`. The Spanish translation comes from the USAGov en Español team.

## Major changes

n/a

## Testing and review

1. Open the Identifier in Storybook
1. Confirm that for each English-language variant, the accessibility link reads "Accessibility statement"
1. Confirm that for each Spanish-language variant, the accessibility link reads "Declaración de accesibilidad"
